### PR TITLE
Use UBI containers in e2e tests

### PR DESCRIPTION
**Pull request by katrybacka**

### CODE REVIEW

File: test/env/update_local_registry.sh [hunk 1]
```diff
@@ -19,43 +19,43 @@ contrail-operator/engprod-269421/contrail-operator:master.latest
...
+contrail-nightly/contrail-kubernetes-kube-manager:2011.55-ubi
+contrail-nightly/contrail-kubernetes-cni-init:2011.55-ubi
+contrail-nightly/contrail-node-init:2011.55-ubi
+contrail-nightly/contrail-nodemgr:2011.55-ubi
+contrail-nightly/contrail-controller-config-dnsmasq:2011.55-ubi
+contrail-nightly/contrail-analytics-api:2011.55-ubi
+contrail-nightly/contrail-analytics-collector:2011.55-ubi
+contrail-nightly/contrail-analytics-query-engine:2011.55-ubi
+contrail-nightly/contrail-controller-config-devicemgr:2011.55-ubi
 contrail-nightly/contrail-command:2011.2
```
Comment (kaweue) [inline test/env/update_local_registry.sh:59]: Here is command second time without ubi is that expected?

Comment (katrybacka) [reply test/env/update_local_registry.sh:59]: nope, I missed that

Comment (katrybacka) [reply test/env/update_local_registry.sh]: fixed

**Review by kaweue — APPROVED** [review]

**Comment by katrybacka** [comment]
recheck

**Comment by katrybacka** [comment]
recheck ha

**Review by kabart — APPROVED** [review]

**Pull request merged**